Resources/Prototypes/Recipes/Construction/Graphs/window.yml

```diff
@@ -11,7 +11,7 @@
               doAfter: 2
             - material: Glass
               amount: 2
-            - material: Plasma
+            - material: PlasmaGlass
```

Comment:
Two things. First one, I really don't like this change.
Second one, why would you change Plasma to PlasmaGlass and leave Glass unchanged?

Reply:
ok so to make plasma glass you need 1 plasma and 1 glass and to make plasma windows you need plasma not plasma glass just plasma which makes no sense whats the point of plasma glass if it dosen't work for making plasma windows 
![image](https://user-images.githubusercontent.com/84478872/121799553-a4ece200-cbe1-11eb-8e96-03d453a1163d.png)

Reply:
Your new recipe makes no sense, you clearly don't know what you're doing... If you're gonna make this use plasma glass sheets, *remove glass as a material in the recipe*